ecs-agent/utils/netlinkwrapper/netlink_linux.go

```diff
@@ -38,3 +40,11 @@ func (nl *netLink) LinkByName(name string) (netlink.Link, error) {
 func (nl *netLink) LinkSetUp(link netlink.Link) error {
 	return netlink.LinkSetUp(link)
 }
+
+func (nl *netLink) RouteList(link netlink.Link, family int) ([]netlink.Route, error) {
```

Comment:
Why do we need these methods here to be non-static? Seems like the variable `nl` is not used. Should we use `return nl.RouteList(link, family)`?

Reply:
Mainly for testing purpose to have this internal only object.